yourgov/pages/app/staff/index.tsx

```diff
@@ -70,10 +71,49 @@ const Page: NextPageWithLayout = () => {
 											benefits to your business and help streamline your
 											operations.
 										</Text>
-
-										<TextLink>
-											Learn more about inviting a staff member
-										</TextLink>
+										<HelpReference
+											drawerContent={
+												<Stack gap={2}>
+													<H1>Learn more about inviting a staff member</H1>
```

Comment:
Interesting that you've gone for a `h1` here. I thought [this was advised against](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements#avoid_using_multiple_h1_elements_on_one_page). What are your thoughts?

Reply:
Interesting. I'm sure the real HelpReference component just pulls out the content of the page to plonk in the drawer, and since the heading is an h1, that's what gets rendered. I feel it's ok in this scenario though, because it is an h1. It's the heading of this page, not a sub heading of the parent page. Being in a dialog means you can't access the other either. Will be good to test